pbm/slicer/slicer.go

```diff
@@ -155,7 +155,6 @@ func (s *Slicer) Catchup(ctx context.Context) error {
 		err = s.copyReplsetOplog(ctx, rs)
 		if err != nil {
 			s.l.Error("copy oplog from %q backup: %v", lastBackup.Name, err)
-			return nil
```

Comment:
This change will produce oplog hole within PITR chunks for the whole duration of the backup procedure.